### ISSUE TITLE
Problem with errorOptions with Vuetify components

### ISSUE THREAD

**Issue by AlexanderStoffers** [jarvelov/vue-form-json-schema#32]
Hey there,

i started using vue-form-json-schema with Vuetify components.

After all it´s working quite well, but we are still struggling with validation on load:

We would like to to validate data on load and display validation errors using "error" and "error-message" props in the Vuetify component.

We use these options:
` options: {
        showValidationErrors: true,
        validate: true,
        validateOnLoad: true
      }`

We use that ui schema:
`
              {
                "component": "v-select",
                "model": "type",
                "errorOptions": {
                  "attrs": {
                    "error": true,
                    "error-messages": "This field is required"
                  }
                },
                "fieldOptions": {
                  "class": ["xs12"],
                  "on": ["input"],
                  "attrs": {
                    "placeholder": "Please select a audit type",
                    "label": "Audit Type",
                    "disabled": false,
                    "readonly": false,
                    "items": ["Audit 1", "Audit 2", "Audit 3"]
                  }
                }
              }
`

After all the validation error is only displayed if i touched the component in the UI. 

Do you have a hint how to deal with that problem?

Best
Alex

**Comment by jarvelov** [comment]
Hi Alex,

Thank you for reporting this. It seems like a bug.
I've replicated the error here: https://codesandbox.io/s/example-1-49syi

I think I know what causes this error and I think I know how to fix it. I'll investigate when I get home and report back.

**Comment by erpnextdeveloper2** [comment]
Hello Alex,

Can you please share how you configured this with the Vuetify library?
So how did you achieved to display Input Styles of Vuetify using this library?

Thanks!

**Comment by erpnextdeveloper2** [comment]
[Vue warn]: Unknown custom element: <v-select> - did you register the component correctly? For recursive components, make sure to provide the "name" option.

I get this error while i add v-select component in my form

**Comment by jarvelov** [comment]
@erpnextdeveloper2 Please see this example: 

Specifically for the styles you have to add `import "vuetify/dist/vuetify.min.css";` as seen in `App.vue`.
You get an unknown custom element because you haven't registered the component globally. Your `main.js` (or equivalent) should look like below:

```js

import Vue from "vue";
import Vuetify from "vuetify";
import App from "./App";

Vue.config.productionTip = false;

Vue.use(Vuetify);

/* eslint-disable no-new */
new Vue({
  el: "#app",
  components: { App },
  vuetify: new Vuetify({}),
  template: "<App/>"
});
```

**Comment by jarvelov** [comment]
@AlexanderStoffers I've fixed the issue in https://github.com/jarvelov/vue-form-json-schema/tree/fix/validate-on-load, but I need to address some performance concerns before I can publish it. I should have the fix published by tomorrow in `v2.3.7`.

**Comment by jarvelov** [comment]
The error has now been fixed and has been published in the next minor version, `v2.4.0`. The `v2.3.7`was skipped due to the inclusion of another feature #17

You can see a working example here: https://codesandbox.io/s/example-1-efvh9

**Comment by AlexanderStoffers** [comment]
Wow,

thank you for your really fast response!

I just had a look in your example an that was just the behaviour i expected.

I´m just testing the updated version regarding the problems reported in issue #34 


Best Regards
Alex

Btw.: We also would appreciate to have an API to manually validate the model due to performance improvment :-)

**Comment by jarvelov** [comment]
Hi Alex,

Thank you for confirming the expected behavior. I hope to have a fix out tonight for #34. I'll inform you here when it is published.

> Btw.: We also would appreciate to have an API to manually validate the model due to performance improvment :-)

I hear you, I have thought about exposing some vfjs methods but never gotten around to it. I'll create a roadmap where I can follow up on this.

**Comment by jarvelov** [comment]
@AlexanderStoffers

I've just released v2.5.0 in which the irregular validation behavior has been patched. Can you try the new version and see if you encounter any irregularities?

Also, I am working on making some vfjs methods available, like validate. I'll keep you informed.
